assets/src/components/notificationDrawer.tsx

```diff
@@ -43,79 +46,36 @@ const TitleBar = () => {
 }
 
 const Content = () => {
-  const { notifications } = useContext(NotificationsContext)
+  const { dispatch, notifications } = useContext(NotificationsContext)
   const currentTime = useCurrentTime()
 
   const [, stateDispatch] = useContext(StateDispatchContext)
 
+  const notificationsDispatch = useContext(NotificationsContext).dispatch
```

Comment:
Am I missing something or is `notificationsDispatch` the same value as `dispatch` (but distinct from `stateDispatch`)?

Reply:
whoops...so it is.